Toolbar1/toolbarplugin.js

```diff
@@ -3,4 +3,4 @@ ToolbarPlugin.Button = function()
 {
     console.log("Toolbar Plugin button clicked!!");
 }
-FormIt.Commands.RegisterJSCommand("ToolbarPlugin.Button");
```

Comment:
This will prevent assigning a shortcut for the command.  What is wrong with this line?

Reply:
Actually, this change will prevent the burn click from doing anything.

Reply:
The problem is, this line is being recognized as Formit Api rather than custom function, and the plugin is trying to invoke it as Formit Api so want to try out the code without the register command